demo-playground/Run.hs

```diff
@@ -46,7 +44,12 @@ runNode CLI{..} = do
          SimpleNode t            -> handleSimpleNode t
 
 -- The concrete block this demo will run with.
-type Block = Mock.SimpleBlock (Bft BftMockCrypto) Mock.SimpleBlockStandardCrypto
```

Comment:
Let's move this to `SimpleBlockMockCrypto`, as part of the mock ledger definition itself, for consistency with everything else.